sms/v1/openapi.yaml

```diff
@@ -81,6 +100,146 @@ paths:
+            they are displayed as one message (if supported by the handset).
+
+            `tyntec will charge for each part of the concatenated message as an individual message.`
+
+          required: true
+          schema:
+            type: string
+        - name: encoding
+          in: query
+          description: Encoding selection between GSM7, UNICODE or the default AUTO selection
+          schema:
+            type: string
+            enum:
+              - AUTO
+              - GSM7
+              - UNICODE
+            default: AUTO
+        - name: gateway
+          in: query
+          description: Which gateway to use for the delivery of the message
+          schema:
+            type: string
+            default: DEFAULT
+        - name: conversion
+          in: query
+          description: Is this message subject to conversion ratio
+          schema:
+            type: boolean
+            default: false
+        - name: sendDateTime
+          in: query
+          description: >-
+            Any future date in the format “yyyy-MM-ddT-HH:mm:ss+HH:mm” [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+            If not set, message will be sent immediately.
+            The default time zone is [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time)
+          schema:
+            type: string
+        - name: validity
+          in: query
+          description: Validity in minutes for this message
+          schema:
+            type: string
+            default: '2880'
+        - name: reference
+          in: query
+          description: A custom reference that will mark the Delivery Report
+          schema:
+            type: string
+        - name: callbackUrl
+          in: query
+          description: Your URL for delivering the Delivery Reports. Leave empty for no Delivery Report
+          schema:
+            type: string
+        - name: callbackMethod
+          in: query
+          description: Your prefered HTTP method for the Delivery Report callback. Possible values POST/GET
+          schema:
+            type: string
+        - name: partLimit
+          in: query
+          description: Up to how many parts you allow this message to be concatenated
+          schema:
+            type: integer
+            format: int32
+            default: 4
+        - name: trate
+          in: query
+          description: >-
+            Up to how much you allow the transcoder to replace original
+            characters to make it [GSM-7](http://en.wikipedia.org/wiki/GSM_03.38)
+            compatible in order for it to fit in less parts.
+            Possible decimal values from 0 to 1. (0.5 would mean 50% or the original content can be replaced)
+          schema:
+            type: number
+            format: double
+            default: 1
+        - name: mrate
+          in: query
+          description: >-
+            Up to how much you allow the transcoder to skip original
```

Comment:
Controls the replacement rate of characters incompatible with [GSM-7](http://en.wikipedia.org/wiki/GSM_03.38). They're replaced with '.'.